arch/risc-v/src/common/supervisor/riscv_dispatch_syscall.S

```diff
@@ -91,11 +91,6 @@ riscv_dispatch_syscall:
   addi       s0, sp, XCPTCONTEXT_SIZE
```

Comment:
Here save_ctx makes a bit of extra work now due to how you have merged the status / epc handling into save_ctx, same with load_ctx